net.go

```diff
@@ -0,0 +1,27 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/shirou/gopsutil/net"
+)
+
+func netInfo() {
+	a, _ := net.Interfaces()
+
+	fmt.Println()
+	fmt.Println("\t\t\tNetwork Info")
+
+	netSpeed, err := exec.Command("ethtool", "eno16777984").Output()
```

Comment:
So we want to avoid calling out to exec... The reason being, are you guaranteed that on every machine that sqrl runs on you will have "ethtool" available?  We need to see if there is a pkg that can get us the information we need.  See if you can't find one, otherwise we will have to regroup and figure something out.